hoomd/BondedGroupData.h

```diff
@@ -252,11 +263,14 @@ class BondedGroupData
 
     virtual ~BondedGroupData();
 
+    //! Initialize internal memory
+    void initialize();
+
     //! Initialize from a snapshot
-    virtual void initializeFromSnapshot(const Snapshot& snapshot);
```

Comment:
These should remain virtual. Future subclasses may need to override these methods.